Helper/Data.php

```diff
@@ -1067,7 +1067,10 @@ public function isPaymentMethodOpenInvoiceMethod($paymentMethod)
         if (strpos($paymentMethod, 'afterpay') !== false ||
             strpos($paymentMethod, 'klarna') !== false ||
             strpos($paymentMethod, 'ratepay') !== false ||
-            strpos($paymentMethod, 'facilypay_') !== false
+            strpos($paymentMethod, 'facilypay_') !== false ||
+            strpos($paymentMethod, 'affirm') !== false ||
+            strpos($paymentMethod, 'clearpay') !== false ||
```

Comment:
@cyattilakiss I've been testing and could Clearpay code be `afterpaytouch` instead?

Reply:
@acampos1916 afterpaytouch should be covered already with the strpos($paymentMethod, 'afterpay') !== false part.
Why?

Reply:
Just that I was working with Clearpay and found out that the PM code for that is `afterpaytouch`, but you're right indeed, it is a strpos